Assignment/solution_student/aboutme.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>About Me</title>
+	<link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+
+	<nav> 
+		<a href="aboutme.html">About Me</a> |
+		<a href="portfolio.html">Portfolio</a>
+	</nav>
+
+	<h1><center>JOE JOHNSON</center></h1>
+
+<hr>
+
+	<img src="images/avatar.png" alt="avatar image" align="right" class= "avatar image-border">
+
+	<h4 class= "letter spacing">I ENJOY LIFE AS A DEVELOPER</h4>
```

Comment:
You added a space between the "=" and the opening quotes, which is why the classes didn't work here:
`class="letter-spacing"` (I also assumed you meant `letter-spacing`)